scripts/bootstrap.js

```diff
@@ -21,4 +21,11 @@ const ciCommand = npmCmd === 'ci'
 const commandArgs = ciCommand ? '--ci' : '--no-ci'
 const environmentArgs = ciEnvironment ? '--concurrency 1 --loglevel debug' : ''
 const cmd = `npm run lerna -- bootstrap ${commandArgs} ${environmentArgs}`
+// Symlink local dependencies
+// A bug manifests itself on Windows where lerna bootstrap fails to generate the `pwa-kit-cli` bin shims
+// in the bin file. As a result, Windows runs the `pwa-kit-cli` bin file using Windows Host Script instead
+// of Node. We run the lerna link command to fix the shims in the `pwa-kit-cli` bin file on Windows.
+if (process.platform === 'win32') {
+  childProc.execSync('npm run lerna link --force-local', {stdio: 'inherit'})
+}
```

Comment:
The change doesn't fix the issue.

**Steps to reproduce:**
1. Checkout the branch `v2-add-readme-note-for-windows` on Windows
2. Run `git clean -dxf && npm ci`
3. Move to a template folder `cd packages\template-express-minimal`
4. Run `npm run lint`
5. Observe the Windows Script Host error